src/main/java/org/cyclops/integratedtunnels/part/PartStateItem.java

```diff
@@ -39,6 +39,6 @@ public ItemStack extractItem(int slot, int amount, boolean simulate) {
 
     @Override
     public int getSlotLimit(int slot) {
-        return getPositionedAddonsNetwork() != null ? getPositionedAddonsNetwork().getSlotLimit(slot) : null;
```

Comment:
`getPositionedAddonsNetwork()` can in fact return `null`. The `null` return value should just be replaced with `0` I assume?